IndexedDB/remove-listener-in-handler.htm

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>removeEventListener in a handler doesn't interfere with future transactions</title>
+<link rel="author" title="quasicomputational" href="https://github.com/quasicomputational">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+<script>
+const t = async_test();
+
+const STORE = "store";
+
+let count = 0;
+
+const go = () => {
+    const dbReq = indexedDB.open("db", 1);
```

Comment:
Please include an `add_cleanup` call to delete the database, and consider using helpers from `support.js` to generate unique names and do pre/post test cleanup, so that this test doesn't start failing if run after another test (e.g. that also fails to clean up)